version/10_12_0.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
-NODE_VERSION=10.12.0
+NODE_VERSION=10.13.0
```

Comment:
Same problem as before of file name not matching the version. I think the files should be called `10_x.sh`, `11_x.sh`, etc., so we don't have to keep renaming them.

Reply:
ok